docs/overview/learning-path-beginner.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Start Here! A guide for beginners making Office Add-ins
+description:  A recommended path for beginners through the learning resources for Office Add-ins.
+ms.date: 02/19/2020
+ms.topic: conceptual
+ms.custom: scenarios:getting-started
+localization_priority: Priority
+---
+
+# Start Here! A guide for beginners making Office Add-ins
+
+Thank your for your interest in Office Add-ins, the Office extensions that run cross-platform, including Office on the web, Windows, and Mac. Like many people who are exploring a new framework, your first question might be **"Where do I start?"**. So, we created this recommended sequence of resources.
+
+## 1 Begin with fundamentals
+
+We know you're eager to start coding, but there are some things about Office Add-ins that you should read before you open your IDE or code editor.
```

Comment:
In my user tests, most people do what you are cautioning them not to do here (code right away). I just think if it were me and someone said "wait wait don't code yet" I would perversely want to do that even more than I already would. 
```suggestion
Learn the essentials of what an Office Add-in is, with these short overview topics: 
```

Reply:
Style convention is 
"Step 1: Begin with fundamentals"
"Step 2: Blah and blah in the blah blah"
etc. Otherwise, omit numbering!

Reply:
done